packages/pangea-sdk/.sdk-ci.yml

```diff
@@ -25,6 +25,7 @@
     SERVICE_VAULT_ENV: LVE
     SERVICE_SHARE_ENV: LVE
     SERVICE_SANITIZE_ENV: LVE
+    SERVICE_AUTHZ_ENV: DEV
```

Comment:
We could point it directly to LVE, it's already available there.

Reply:
Done.